02-repositories/redpanda.yaml

```diff
@@ -0,0 +1,3 @@
+name: pulumi-redpanda
+description: Pulumi provider for Redpanda Cloud
+type: provider
```

Comment:
Please add `import: true` because the repository already exists.

Reply:
added!